Dockerfile.prod

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8.5-buster
```

Comment:
shouldn't there be a single Dockerfile? 
I would hope so, otherwise, how can we test it? OR rather, we have then two products that need to be tested.